IPython/core/display.py

```diff
@@ -512,31 +512,54 @@ def _repr_svg_(self):
 
 
 class JSON(DisplayObject):
-    """JSON expects a JSON-able dict or list
-    
-    not an already-serialized JSON string.
-    
-    Scalar types (None, number, string) are not allowed, only dict or list containers.
-    """
+
```

Comment:
Sure, I can do that! My thinking was to make it similar to the `Image` class, which is why I removed the class doc string. I'll add it back though. Thanks!